client/state/plugins/upload/actions.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Internal dependencies
+ */
+import {
+	PLUGIN_UPLOAD,
+	PLUGIN_UPLOAD_UPDATE_PROGRESS,
+	PLUGIN_UPLOAD_COMPLETE,
+	PLUGIN_UPLOAD_ERROR,
+} from 'state/action-types';
+
+/**
+ * Upload a plugin to a site.
+ *
+ * @param {number} siteId site ID
+ * @param {File} file the plugin zip to upload
+ * @return {Object} action object
+ */
+export function uploadPlugin( siteId, file ) {
```

Comment:
It's usually one test per action creator, and it doesn't give too much value, but it still tests for the action structure that we'll be relying on, so it makes sense for it to be tested IMO. I'll leave it to your judgement.